### DISCUSSION TITLE
New to lazyvim -- autocompletion question

### DISCUSSION THREAD

**Discussion by willij74** [LazyVim/LazyVim#413] [General]
Hey everyone. I just started to use lazyvim and quite like it. I have used vim for sometime now and just starting to learn/transition into using the lua side, which is new to me

Hopefully, a simple question.
Is it possible, to control what type of files or languages are part of autocompletion?
For example, I do a lot of work in javascript, go, yaml as well as markdown.

As odd as it may sound, I would like to disable any recommendations for standard words when writing in markdown. I tend to be a heavy markdown writer and prefer not to have drop down/auto completion enabled.

I am very new to this, so I am still learning as I go.
Really appreciate the feedback and help!

Cheers,

W

**Comment by folke** [comment]
Depends on the configuration of your lsp and on the sources you added to cmp. 

Check the cmp docs and your markdown lsp (if you have on) to see how you can configure it to disable what you want.

**Comment by markstos** [comment]
This spec disables plugins that cause completion in Markdown. It disables them for all other filetypes as well, but it may help set you on the path towards creating a config that works better for you:

```lua
return {
  {
  	"L3MON4D3/LuaSnip",
    enabled = false,
  },
  {
    "rafamadriz/friendly-snippets",
    enabled = false,
  },
  {
    "hrsh7th/cmp-buffer",
    enabled = false,
  }
}

```
